packages/grafana-ui/src/components/uPlot/Plot.tsx

```diff
@@ -117,12 +141,150 @@ export class UPlotChart extends React.Component<PlotProps, UPlotChartState> {
     }
   }
 
+  handleKeyRelease = (e: React.KeyboardEvent) => {
+    if (!SUPPORTED_KEYS.includes(e.key as SupportedKey)) {
+      return;
+    }
+
+    this.keyboardState.pressedKeys[e.key as SupportedKey] = false;
```

Comment:
As LOVELY as the SUPPORTED_KEYS and the derived SupportedKey types are, I don't think they're actually giving us any type safety, and we end up having to jump through hoops (like this type assertion, which I would like to completely eliminate from our code) because of it. I do appreciate that this is the safest unsafe code though 😎 

I think it's completely fine for `state.pressedKeys` to be typed to `Record<string, boolean>`, because that's basically what it is anyway. I don't think there's that many bugs that we would catch because of the `SupportedKey` type.